src/components/InputTooltip.js

```diff
@@ -104,34 +135,45 @@ class InputTooltip extends Component {
                 .then(({ suggestions }) => {
                     this.setState(prevState => ({
                         suggestions,
-                        cachedResults: { ...prevState.cachedResults, [query]: suggestions },
+                        cachedResults: {
+                            ...prevState.cachedResults,
+                            [query]: suggestions,
+                        },
                     }));
-                })
-                .catch(() => this.setState({ isError: true }));
+                });
         }
     }
 
-    accept(url) {
-        const { onAccept } = this.props;
-        const trimmedUrl = url.trim();
-
-        if (this.isValidUrl(trimmedUrl)) {
-            onAccept(trimmedUrl);
-        } else {
-            this.setState({ isLinkInvalid: true });
-        }
+    setSuggestionsListClicked(value) {
+        this.suggestionsListClicked = value;
     }
 
     splitText(textToSplit, query) {
-        const escapedQuery = query.trim().toLowerCase().replace(/[-/\\^$*+?.()|[\]{}]/g, '\\$&');
-        const matchedIndex = textToSplit.toLowerCase().indexOf(escapedQuery);
+        const escapedQuery = query.trim()
+            .toLowerCase()
+            .replace(/[-/\\^$*+?.()|[\]{}]/g, '\\$&');
```

Comment:
:beer: if you can tell me what it matches in months 😏🍻 can you add a comment please?